_search-plugins/knn/knn-index.md

```diff
@@ -112,6 +114,38 @@ The IVF algorithm requires a training step. To create an index that uses IVF, yo
 data points, but it is [recommended to use more](https://github.com/facebookresearch/faiss/wiki/Guidelines-to-choose-an-index#how-big-is-the-dataset).
 Training data can either the same data that is going to be ingested or a separate set of data.
 
+### Supported Lucene methods
+
+Method Name | Requires Training? | Supported Spaces | Description
+:--- | :--- | :--- | :---
+`hnsw` | false | "l2", "innerproduct", "cosinesimil", "l1", "linf" | Hierarchical proximity graph approach to Approximate k-NN search.
```

Comment:
Are quotation marks necessary here and below?

Reply:
Good question. This is the way they are expressed in JSON for configuration. For now, I've removed quotes around all instances. May be open for editorial review.